stickynotes/stickynotes.c

```diff
@@ -948,10 +951,12 @@ stickynotes_load (GdkScreen *screen)
     xmlDocPtr doc = NULL;
     xmlNodePtr root;
     xmlNodePtr node;
-    /* WnckScreen *wnck_screen; */
     GList *new_notes, *tmp1;  /* Lists of StickyNote*'s */
     GList *new_nodes; /* Lists of xmlNodePtr's */
     int x, y, w, h;
+#ifdef GDK_WINDOWING_X11
+    GdkDisplay *display = gdk_screen_get_display(gdk_screen_get_default());
```

Comment:
```suggestion
    GdkDisplay *display = gdk_screen_get_display (gdk_screen_get_default());
```

Reply:
Done

Reply:
Ahh, more of them, will get them now and push again

Reply:
done